migrations/0012_add_sku_to_order_item.up.sql

```diff
@@ -0,0 +1,2 @@
+alter table order_items
+add sku text;
```

Comment:
should we make sku non-nullable?

Reply:
https://github.com/brave-intl/bat-go/pull/313/commits/9ec1d0a47d9bea579f735442b2f250a017bbb9b9